conductor/blueprints/authorization/controllers.py

```diff
@@ -69,21 +72,9 @@ def __call__(self):
         return PUBLIC_KEY
 
 
-class Sample:
```

Comment:
@pwalsh you're right))

`__call__` here is reasonable because it's a controller interface implementation like - https://github.com/openspending/os-conductor/blob/feature/datastore-authorization/conductor/blueprints/datastore/controllers.py

Because controller could have a state we use classes here instead of functions. But function also could be for situations for state-less controllers like `Lib`.